lib/cmds/setup.js

```diff
@@ -80,6 +80,26 @@ exports.handler = async argv => {
       mask: '*',
       when: a => a.useClientCredentials,
       default: a => config.get(`${a.environment}.defaults.clientSecret`)
+    },
```

Comment:
Did you try using the `https_proxy` or `http_proxy` environment variables?   Axios supports using them.  I just wonder if that's a simpler way to go rather than add in this proxy configuration everywhere.

Reply:
I tried using them - no success. This is the only things that dependably worked. Looking at threads like this one also confirmed that https over proxy is even trickier and had to code it explicitly - we could consider using it passively from environment variables versus configurations, but looks like Axios isn't correctly handling it or it doesn't work with our proxy setup. I just re-confirmed it doesn't work just with your master lo client using the environment variables.